docs/api-guide/views.md

```diff
@@ -217,11 +217,26 @@ You may pass `None` in order to exclude the view from schema generation.
     def view(request):
         return Response({"message": "Will not appear in schema!"})
 
+# Async Views
+
+When using Django 4.1 and above, REST framework allows you to work with async class based and function based views.
```

Comment:
class-based
function-based

Reply:
I think you can remove "based" altogether for brevity.

Reply:
Removed. Thanks.